code/defines/procs/statistics.dm

```diff
@@ -55,70 +52,34 @@
 
 	var/turf/T = H.loc
 	var/area/placeofdeath = get_area(T.loc)
-	var/podname = sanitizeSQL(placeofdeath.name)
+	var/podname = placeofdeath.name
 
-	var/sqlname = sanitizeSQL(H.real_name)
-	var/sqlkey = sanitizeSQL(H.key)
-	var/sqlpod = sanitizeSQL(podname)
-	var/sqlspecial = sanitizeSQL(H.mind.special_role)
-	var/sqljob = sanitizeSQL(H.mind.assigned_role)
-	var/laname
-	var/lakey
+	// Empty string is important here!
+	var/laname = ""
+	var/lakey = ""
 	if(H.lastattacker)
-		laname = sanitizeSQL(H.lastattacker)
+		laname = H.lastattacker
 	if(H.lastattackerckey)
-		lakey = sanitizeSQL(H.lastattackerckey)
-	var/sqltime = time2text(world.realtime, "YYYY-MM-DD hh:mm:ss")
-	var/coord = "[H.x], [H.y], [H.z]"
-//	to_chat(world, "INSERT INTO death (name, byondkey, job, special, pod, tod, laname, lakey, gender, bruteloss, fireloss, brainloss, oxyloss) VALUES ('[sqlname]', '[sqlkey]', '[sqljob]', '[sqlspecial]', '[sqlpod]', '[sqltime]', '[laname]', '[lakey]', '[H.gender]', [H.bruteloss], [H.getFireLoss()], [H.brainloss], [H.getOxyLoss()])")
-	establish_db_connection()
-	if(!GLOB.dbcon.IsConnected())
-		log_game("SQL ERROR during death reporting. Failed to connect.")
-	else
-		var/DBQuery/query = GLOB.dbcon.NewQuery("INSERT INTO [format_table_name("death")] (name, byondkey, job, special, pod, tod, laname, lakey, gender, bruteloss, fireloss, brainloss, oxyloss, coord) VALUES ('[sqlname]', '[sqlkey]', '[sqljob]', '[sqlspecial]', '[sqlpod]', '[sqltime]', '[laname]', '[lakey]', '[H.gender]', [H.getBruteLoss()], [H.getFireLoss()], [H.getBrainLoss()], [H.getOxyLoss()], '[coord]')")
-		if(!query.Execute())
-			var/err = query.ErrorMsg()
-			log_game("SQL ERROR during death reporting. Error : \[[err]\]\n")
-
-//This proc is used for feedback. It is executed at round end.
-/proc/sql_commit_feedback()
```

Comment:
This proc was never called ever. It was also very confusing given the existing blackbox feedback code. It has been purged for good reason.